internal/distro/fedora/distro.go

```diff
@@ -856,6 +858,12 @@ func (t *imageType) checkOptions(customizations *blueprint.Customizations, optio
 		return nil, err
 	}
 
+	repos := customizations.GetRepositories()
+	err = blueprint.ValidateCustomRepositories(repos)
```

Comment:
I think that it would be better to validate when we unmarshal from TOML, but this is also OK.